eos_data_distribution/ndn/file.py

```diff
@@ -104,9 +104,6 @@ class Consumer(chunks.Consumer):
     def __init__(self, *args, **kwargs):
         super(Consumer, self).__init__(*args, **kwargs)
 
-    # XXX: This is disgusting hackery. We need to remove auto=True.
-
-    def start(self):
```

Comment:
Scheduling the interests should still remain in start(), just move the code that reads the segment table.